LaunchDarklyAndroidBinding/LaunchDarklyAndroidBinding.csproj

```diff
@@ -46,20 +46,24 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Java.Interop" />
     <Reference Include="GoogleGson">
-      <HintPath>\tmp\nuget\GoogleGson.2.8.1\lib\MonoAndroid\GoogleGson.dll</HintPath>
+      <HintPath>..\packages\GoogleGson.2.8.1\lib\MonoAndroid\GoogleGson.dll</HintPath>

```

Comment:
Sorry, just noticed this as I was approving - can we change these back to `\tmp\nuget` path references? That helps a little with not duplicating the same nuget packages in multiple locations across multiple repos

Reply:
^^